### PR TITLE
Java SDK

### PR DESCRIPTION
**Pull request by rphumulock**
- Includes a server under /Examples directory.
- SDK code and SDK hosted on fork packages. 

We can talk about where to host the SDK. If anyone notices anything weird just let me know.

### CODE REVIEW

File: sdk/java/README.md [hunk 1]
```diff
@@ -0,0 +1,155 @@
+Here's a `README.md` for your **Datastar Java SDK** following the format you provided:
```
Comment (delaneyj) [inline sdk/java/README.md]: maybe remove this

**Review by delaneyj — CHANGES_REQUESTED** [review]
You also need to add yourself to the `build/run.go` generated files for consts and defaults.  Look at the other `consts_{lang}.qtpl` example.  If you need help reach out.

**Comment by bencroker** [comment]
Please move the `examples` folder into `/sdk/java`.

**Comment by delaneyj** [comment]
I moved the dotnet examples to examples, @rphumulock was following my lead here

**Pull request merged**